src/core/server/deprecations/README.mdx

```diff
@@ -0,0 +1,194 @@
+---
+id: kibUsageCollectionPlugin
+slug: /kibana-dev-docs/services/deprecations-service
+title: Core Deprecations service
+summary: The Deprecations service helps surface deprecated configs and features for plugins to our users
+date: 2021-05-03
+tags: ['kibana','dev', 'contributor', 'api docs']
+---
+
+# Core Deprecations service
+This guide is written in the format of a FAQ to help you get started using the deprecations service.
+For more details on the service contract we've documented all the service apis and properties under the deprecation service docs generated by in core.
+
+## What is the deprecations service?
+The deprecations service provides a way for the Kibana platform to communicate deprecated features and configs with its users. These deprecations are only communicated if the deployment is using these features. Allowing for a user tailored experience for upgrading the stack version.
```

Comment:
```suggestion
The deprecations service provides a way for the Kibana platform to communicate deprecated features and configs with its users. These deprecations are only communicated if the deployment is using these features, allowing for a user tailored experience for upgrading the stack version.
```

In the past, I know that the Cloud team has also asked for Kibana to provide an API that signals to Cloud that the Stack is ready for a major upgrade. Is this data also going to be used for that? If so, I think that should be explained.

Reply:
I havent heard about this before. I guess UA would be providing this API rather than core since UA is also exposing elasticsearch deprecations. @alisonelizabeth do you have some context on this?

Reply:
I'm still trying to get more information as to what cloud will need in terms of UA. But I agree that I think UA would likely be responsible for providing an API so that it can account for ES and Kibana deprecations.